api-gateway/src/main/resources/static/scripts/inventory-product-list/inventory-product-list.template.html

```diff
@@ -78,7 +78,28 @@ <h2>Inventory Products</h2>
         <td></td>
     </tr>
 </table>
-<a ui-sref="productNew({ inventoryId: $ctrl.inventoryProductList[0].inventoryId })">
+<div class="d-flex justify-content-center pb-3">
+    <nav aria-label="Page navigation example">
+        <ul class="pagination">
+            <li class="page-item">
+                <a class="page-link text-dark" aria-label="Previous" ng-click="$ctrl.previousPage()">
+                    <span aria-hidden="true">&laquo;</span>
+                </a>
+            </li>
+
+            <li class="page-item">
+                <a class="page-link text-dark" disabled>Page-{{$ctrl.actualCurrentPageShown}}</a>
```

Comment:
I would also like to add, should the pagination button CSS is identical or similar to the inventory button look-wise. So that it matches and makes it look better.

Reply:
i just didn't like the look of the inventory one